src/future/future.rs

```diff
@@ -99,6 +107,28 @@ extension_trait! {
     }
 
     pub trait FutureExt: std::future::Future {
+        /// Creates a future that is delayed before it starts yielding items.
```

Comment:
Looking at the implementation, I'm not sure if this documentation is correct.

To my reading, it delays _execution_ of the future for the time time given. It will never be polled before the delay resolves. (which might or might not be what you want)

Reply:
fixed. I'm not fluent in English, so it might still be wrong ...